invenio_base/urls/builders.py

```diff
@@ -49,26 +49,67 @@ def __init__(
         cfg_of_other_app_prefix,
         groups_of_other_app_entrypoints,
     ):
-        """Constructor."""
+        """Constructor.
+
+        The ``cfg_of_app_prefix`` and ``cfg_of_other_app_prefix`` are the names of
+        config items containing the URL prefixes for both apps (e.g. ``SITE_UI_URL``
+        and ``SITE_API_URL``).
+
+        ``groups_of_other_app_entrypoints`` is expected to be either a dictionary of
+        the shape ``{"blueprints": [...], "converters": [...]}``, containing the names
+        of entrypoint groups.
+        Alternatively the value can be a list, which will be used equivalent to
+        ``{"blueprints": [...]}``.
+        """
```

Comment:
i've taken the liberty to add a bit of documentation about the new interface in a new commit